models/public/nfnet-f0/model.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from timm.models.nfnet import NfCfg, NormFreeNet
+
+
+def create_normfreenet():
+    attn_kwargs = dict(reduction_ratio=0.5, divisor=8)
```

Comment:
#2249 will disallow `dict` calls like this. Please use a dictionary literal instead. I would also suggest to get rid of this variable and just pass the value directly to `NfCfg`.

Reply:
Done